src/VisualStudio/Setup.Next/remoteHostService64.servicehub.service.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "host": "desktopClr",
```

Comment:
In ServiceHub as it is now, the host process will be "ServiceHub.Host.CLR.exe". If you want to have ServiceHub.RoslynCodeAnalysisService* name, we'd need to add appropriate processes in ServiceHub.

Reply:
can you do that?

Reply:
Sure. Bug [#492037](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/492037) will track that.